### PR TITLE
Add public gateway docs

### PR DESCRIPTION
**Pull request by shomix**
This PR will:
- Add a page to the docs for the Spheron Public Gateway

### CODE REVIEW

File: pages/gateways/_meta.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "dedicated": "Dedicated Gateways",
+  "spheron-gateway": "Spheron Gateway"
```
Comment (aayushmahapatra) [inline pages/gateways/_meta.json:3]: Can we name it something like Public Gateway?
Cause even the dedicated gateway is a Spheron Gateway. 🤔

**Review by aayushmahapatra — CHANGES_REQUESTED** [review]
In quick start, update the route for `Gateways` and link it to `Dedicated gateways`.

File: pages/gateways/_meta.json (hunk 1 above)
Comment (shomix) [reply pages/gateways/_meta.json]: renamed to `Public Gateway`

**Comment by shomix** [comment]
> In quick start, update the route for `Gateways` and link it to `Dedicated gateways`.

done.

**Review by rekpero — CHANGES_REQUESTED** [review]
Small RC

**Review by rekpero — APPROVED** [review]

**Pull request merged**